.env.example.docker

```diff
@@ -0,0 +1,24 @@
+# Current values are testing defaults
+# For production defaults remove everything except keys: APP_KEY, DB_PASSWORD, AWS_KEY, AWS_SECRET
+
+APP_URL=https://hotsapi.net
+APP_ENV=local
+APP_KEY=base64:o3X339oA0GFNqXNpmaE/rBl7M2NDfpBXuroKnAs8w8M=
+APP_DEBUG=true
+APP_LOG_LEVEL=debug
+
+DB_HOST=mysql # this is the name of the compose database service
```

Comment:
I think the comment interferes with the `artisan` migration and I was getting an error until I removed it. It was creating a DSN that looked like the following:

```
mysql:host=mysql # this is the name of the compose database service;port=3306;dbname=hotsapi
```
instead of

```
mysql:host=mysql;port=3306;dbname=hotsapi
```